apis/kubedb/constants.go

```diff
@@ -621,6 +621,9 @@ const (
 	KubeDBZooKeeperRoleName         = "kubedb:zookeeper-version-reader"
 	KubeDBZooKeeperRoleBindingName  = "kubedb:zookeeper-version-reader"
 	ZooKeeperClientPortName         = "client"
+	ZooKeeperNodeRolesCombined      = "controller,broker"
```

Comment:
Generate this using fmt.Sprintf(ZooKeeperNodeRolesBroker,ZooKeeprNodeRolesController)